server/src/main/kotlin/com/xebia/functional/xef/server/http/routes/XefRoutes.kt

```diff
@@ -12,4 +12,5 @@ fun Routing.xefRoutes(logger: KLogger) {
   organizationRoutes(OrganizationRepositoryService(logger))
   projectsRoutes(ProjectRepositoryService(logger))
   tokensRoutes(TokenRepositoryService(logger))
+  assistantRoutes()
```

Comment:
Let's add `logging` to this route and use the logger in relevant parts of the code where needed to at least display calls to the endpoint.

For example:
```kotlin
logger.info("Creating assistant: ${assistant.name}")
...
logger.info("Created assistant: ${assistant.name} with id: ${assistant.id}")
```